packages/extension-commerce-bm-seo/src/utils/shopper-seo-utils.tsx

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2025, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+// Platform Imports
+import Auth from '@salesforce/commerce-sdk-react/auth'
+import {ShopperSeo} from 'commerce-sdk-isomorphic'
+
+// Local Imports
+import {Config} from '../types'
+
+export const getShopperSeoClient = async (locals: Record<string, any>, config: Config) => {
```

Comment:
@bendvc @vmarta wanted to check in if we're okay going forward with this approach of pulling in `ShopperSeo` directly from `commerce-sdk-isomorphic`. This is because we need to make the API call in `getRoutesAsync` which is outside the HOC. On lines 24-31 it also looks like we handle saving the `Auth` in `locals` so we don't recreate it each time. 

I think this means the SEO extension *requires* the `commerceApi` to be defined in the config unlike `chakra-store-locator`